pkg/models/log_entry.go

```diff
@@ -16,22 +16,22 @@ package models
 import "encoding/json"
 
 type LogEntry struct {
-	Level         string   `json:"logLevel"`
-	Labels        []string `json:"labels"`
-	OriginService string   `json:"originService"`
-	Message       string   `json:"message"`
-	Created       int64    `json:"created"`
+	Level         string        `json:"logLevel"`
+	Args          []interface{} `json:"args"`
+	OriginService string        `json:"originService"`
+	Message       string        `json:"message"`
+	Created       int64         `json:"created"`
 }
 
 func (l LogEntry) MarshalJSON() ([]byte, error) {
 	test := struct {
-		Level         *string  `json:"logLevel"`
-		Labels        []string `json:"labels"`
-		OriginService *string  `json:"originService"`
-		Message       *string  `json:"message"`
-		Created       int64    `json:"created"`
+		Level         *string       `json:"logLevel"`
+		Args          []interface{} `json:"args"`
+		OriginService *string       `json:"originService"`
+		Message       *string       `json:"message"`
+		Created       int64         `json:"created"`
```

Comment:
I know it's out of the scope of this PR, but should be great if you add the `,omitempty` tag in those five parameters. The rest of this PR LGTM (if you want to merge without this change).

Reply:
@jduranf I agree with the thought of making that additional change (i.e. related to the `omitempty` tag) _separately_, rather than as part of this PR.

Reply:
Adding that this PR, _as_ it stands, LGTM as well 👍